content/projects/understanding-loops/_index.md

```diff
@@ -246,4 +267,4 @@ get_customers_with_open_baskets(shopping_baskets) //python
 - There can be some global constants to help prevent typos. eg `const DELIVERED="DELIVERED"`.
 - Functions are supposed to return very specific things.
 - All the functions stated above should take in at least one argument, the data array, and should work with similar data.
-- **For those using Javascript:** Make sure all their functions functions are exported correctly. Otherwise, the marking bot won't be able to access their code
\ No newline at end of file
+- **For those using Javascript:** Make sure all their functions functions are exported correctly. Otherwise, the marking bot won't be able to access their code.
```

Comment:
Cause you changed it above shouldn't the script in Javascript be uppercase like `JavaScript` here too

Reply:
"Make sure all their functions functions are exported correctly"?

Please rephrase. Please read over all the things very carefully.